darwin/future/data_objects/team.py

```diff
@@ -51,5 +55,32 @@ class Team(DefaultDarwin):
     # Data Validation
     _slug_validator = validator("slug", allow_reuse=True)(parse_name)
 
+    @staticmethod
+    def from_client(client: Client, team_slug: Optional[str] = None) -> Team:
+        """Returns the team with the given slug"""
+        if not team_slug:
+            team_slug = client.config.default_team
+        return get_team(client, team_slug)
+
 
 TeamList = List[Team]
+
+
+def get_team(client: Client, team_slug: Optional[str] = None) -> Team:
+    """Returns the team with the given slug"""
+    if not team_slug:
+        team_slug = client.config.default_team
+    response = client.get(f"/teams/{team_slug}/")
+    return Team.parse_obj(response)
+
+
+def get_team_members(client: Client) -> Tuple[List[TeamMember], List[Exception]]:
+    response = client.get(f"/memberships")
+    members = []
+    errors = []
+    for item in response:
+        try:
+            members.append(TeamMember.parse_obj(item))
+        except Exception as e:
+            errors.append(e)
+    return (members, errors)
```

Comment:
Think the brackets are unnecessary, although totally stylistic, and your choice on this one.